modules/ossm-config-openshift-monitoring-only.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/metrics/ossm-metrics-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-only_{context}"]
+= Configuring OpenShift Monitoring with Service Mesh
+:context: ossm-config-openshift-monitoring-only
+
+//Need some kind of intro
+//Possible file name may change
+//Possible assembly file may change
+//Assemblies, topic map info needs to be worked out still for 3.0, now that docs has a better idea of what is to be included for 3.0 TP1.
+
+//FIX THE STYLE THINGS ONCE DEV HAS REVIEWED/APPROVED THE CONTENT. First priority is ensuring the steps are in the correct order, and are accurate for 3.0 TP1.
+
+The following steps show how to integrate {SMProductName} with user-workload monitoring.
+
+.Prerequisite
+
+ * Monitoring for user-defined projects is enabled. See "Enabling monitoring for user-defined projects".
+
+.Procedure
+
+. Apply a `ServiceMonitor` object to monitor the Istio control plane:
++
+.Example `ServiceMonitor` object
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: istiod-monitor
+  namespace: istio-system
+spec:
+  targetLabels:
+  - app
+  selector:
+    matchLabels:
+      istio: pilot
+  endpoints:
+  - port: http-monitoring
+    interval: 30s
+    relabelings:
+    - action: replace
+      replacement: "basic-istio-system"
+      targetLabel: mesh_id
+----
+
+. Apply a `PodMonitor` object to collect metrics from Istio proxies:
++
+.Example PodMonitor object
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: istio-proxies-monitor
+  namespace: istio-system
```

Comment:
In the [original docs from 2.X](https://docs.openshift.com/container-platform/4.16/service_mesh/v2x/ossm-observability.html#ossm-integrating-with-user-workload-monitoring_observability) in Step 7, this example YAML had a little number next to it with a note. The reason being that this `PodMonitor` example needs to be applied in every application namespace e.g. when you deploy the `bookinfo` app, this `PodMonitor` yaml should get deployed there as well. We'll need some kind of similar note for these docs as well.

Reply:
There was no note here https://github.com/istio-ecosystem/sail-operator/tree/main/docs#scraping-metrics-using-the-openshift-monitoring-stack so it was not added.

<1> says: Since OpenShift Container Platform monitoring ignores the namespaceSelector spec in ServiceMonitor and PodMonitor objects, you must apply the PodMonitor object in all mesh namespaces, including the control plane namespace.

does "including the control plane namespace" still apply? Or does "including the control plane namespace" get removed for 3.0?